webhook/verify.go

```diff
@@ -54,8 +54,8 @@ func (w *WebHook) CheckValues(prefix string) (errs error) {
 	}
 
 	if w.Main != nil {
-		if w.GetType() != "github" {
-			errs = fmt.Errorf("%s%stype: %q <invalid> (the only webhook type is 'github' currently)\\", utils.ErrorToString(errs), prefix, w.GetType())
+		if w.GetType() != "github" && w.GetType() != "gitlab" {
```

Comment:
I'm not a fan of reusing GetType for the same check.
Could we this or similar please? I think it looks cleaner and is makes it easier to add types in the future
```go
types := []string{"github", "gitlab"}
if !utils.Contains(types, w.GetType()) {
	errs = fmt.Errorf("%s%stype: %q <invalid> (supported types = %s)\\", utils.ErrorToString(errs), prefix, w.GetType(), types)
```

Reply:
I can change this too.

Reply:
Have made the changes.

Reply:
Thanks for doing that. Appreciate the time spent doing the PR and hopefully it'll be useful for you and others 😄